lib/attack.go

```diff
@@ -348,6 +350,8 @@ func (a *Attacker) hit(tr Targeter, name string) *Result {
 		}
 	}()
 
+	tgt.Header.Set("X-Vegeta-Seq", strconv.FormatUint(res.Seq, 10))
+
```

Comment:
I don’t think users should be able to over-ride this header unintentionally, so we should set it after the target is read, which also ensures there’s an http.Header (so we don’t need the allocation).